install.sh

```diff
@@ -592,14 +592,17 @@ getshims() {
 
 killtree $({ set +x; } 2>/dev/null; getshims; set -x)
 
-do_unmount() {
-    awk -v path="$1" '$2 ~ ("^" path) { print $2 }' /proc/self/mounts | sort -r | xargs -r -t -n 1 umount
+do_unmount_and_remove() {
+    awk -v path="$1" '$2 ~ ("^" path) { print $2 }' /proc/self/mounts | sort -r | xargs -r -t -n 1 sh -c 'umount $0 && rm -rf $0'
```

Comment:
Is it _guaranteed_ that `$0` will never contain whitespace? (Otherwise, it would need `"$0"`.)

Reply:
(I can see that in the current calls to it, there are no whitespaces, but could it be that someone would be able to add such an entry?)